### ISSUE TITLE
Crash 

### ISSUE THREAD

**Issue by lasnow** [transitive-bullshit/ffmpeg-gl-transition#36]
```
Press [q] to stop, [?] for help
[swscaler @ 0x77bf6e1000] No accelerated colorspace conversion found from yuv420p to rgb24.
[Parsed_gltransition_0 @ 0x77cebd3ec0] invalid vertex shader
[Parsed_gltransition_0 @ 0x77cebd3ec0] Failed to configure input pad on Parsed_gltransition_0
Error reinitializing filters!
Failed to inject frame into filter network: Operation not permitted
Error while processing the decoded data for stream #1:0
Conversion failed!
```

**Comment by liangzs** [comment]
hi lasnow，may i ask you a quetion.
it's i also build for android,then use  "--cc=$TOOLCHAIN/bin/arm-linux-androideabi-gcc \"
that was wrong,show
 `bin/../lib/gcc/arm-linux-androideabi/4.9.x/../../../../arm-linux-androideabi/bin/ld: error: cannot find -lGLEW
/home/liangzs/android/android-ndk-r14b/toolchains/arm-linux-androideabi-4.9/prebuilt/linux-x86_64/bin/../lib/gcc/arm-linux-androideabi/4.9.x/../../../../arm-linux-androideabi/bin/ld: error: cannot find -lglfw`
had you met it ,thank you!

**Comment by lasnow** [comment]
Yep, if you check my build script in the attached zip file you can see that I replaced `-lglfw` and `-lGLEW` for `-lGLESv2 -lEGL` because the NDK compiler does not find them and I didn't know how to include them

**Comment by liangzs** [comment]
appreciate for your reply! but i compile without --enable-opengl，it built sucessed. but test fail like #35 .
when i add --enable-opengl will show  no opengl  environment  so sad.

**Comment by clickear** [comment]
run cmd
/usr/bin/Xorg -noreset +extension GLX +extension RANDR +extension RENDER -logfile ./xdummy.log -config /etc/X11/xorg.conf :1 &
it work for me!
see https://github.com/clofus/ffmpeg-gl/blob/master/init.sh

**Comment by po1xiao** [comment]
> run cmd
> /usr/bin/Xorg -noreset +extension GLX +extension RANDR +extension RENDER -logfile ./xdummy.log -config /etc/X11/xorg.conf :1 &
> it work for me!
> see https://github.com/clofus/ffmpeg-gl/blob/master/init.sh

请问您编译出安卓下使用的libffmpengso库了吗，可以分享一下吗？搞了三天了，都没成功。
